src/pages/gift/generate/Landing.js

```diff
@@ -16,12 +15,12 @@ export default function Landing () {
         <Row className="justify-content-center align-items-center">
           <Col className="d-flex flex-column justify-content-around align-items-center">
             <div className="pt-5">
-              <Button variant="outline-primary" onClick={() => nextStep()}>
+              <button className='btn btn-primary btn-lg' onClick={() => nextStep()}>
```

Comment:
The button in generate landing page is set to be large, but the "Create new account" button in claim page is sized regular, I believe their size should be consistent, either lg or regular.

Reply:
You're right!